tools/clang/test/HLSLFileCheck/hlsl/intrinsics/helper/IsHelperLane.hlsl

```diff
@@ -277,7 +277,7 @@ RWStructuredBuffer<float4> SB;
 void cs(uint gidx : SV_GroupIndex)
 {
   float4 result = a + IsHelperLane();
-  SB[gidx] = QuadReadAcrossX(result);
```

Comment:
curious why this is needed

Reply:
Feature was not allowed by 1.4 validator.

Reply:
ah yes. QuadRead in compute shader.